lib/sqlalchemy/dialects/oracle/cx_oracle.py

```diff
@@ -919,9 +932,11 @@ def output_type_handler(
                         size,
                         cursor.arraysize,
                         outconverter=outconverter,
+                        encodingErrors=dialect.text_encoding_errors
                     )
                 else:
-                    return cursor.var(util.text_type, size, cursor.arraysize)
+                    return cursor.var(util.text_type, size, cursor.arraysize,
```

Comment:
this will fail for cx_Oracle < 6.4 right?  we can't send the flag at all if it isn't in the API.